Docs page navigation adjustments

insectengine opened the pull request:
The docs page had problems with the on page subnav rendering with the added styles. I removed the stickiness to make it present cleaner and created a subnav pulldown for mobile sizes.

insectengine committed:
doc page adjustments. Add on page pulldown navigation button for mobile and adjust content column widths to match wrapper.

aureamunoz commented on docs/_sass/position.scss:
```diff
@@ -179,7 +179,7 @@ $position-large-margin-l:                               50px !default;
 .uk-position-absolute { position: absolute !important; }
 
-.uk-position-fixed { position: fixed !important; }
+.uk-position-fixed { /* position: fixed !important; */ }
```
Shouldn't we just remove the commented part? 
```
.uk-position-fixed { /* position: fixed !important; */ }
```

aureamunoz commented on docs/_sass/width.scss:
```diff
@@ -192,7 +192,8 @@ $width-2xlarge-width:                            750px !default;
-    .uk-width-1-4\@s { width: 25%; }
+/*  .uk-width-1-4\@s { width: 25%; } */
+/*  .uk-width-1-4\@s { width: 25%; } */
     .uk-width-3-4\@s { width: 75%; }
```
Same remark, can we remove the commented part or it is used for something?

aureamunoz reviewed (commented):
Great Job, Thank you!!!! I don't have to much to day, just a question about the commented lines :-)

insectengine commented:
Hi Aurea,

Yes you can remove. I commented it out during testing and should have removed it myself.

Regards
James



> On Jan 18, 2022, at 7:24 AM, Aurea Muñoz Hernández ***@***.***> wrote:
> 
> 
> @aureamunoz commented on this pull request.
> 
> Great Job, Thank you!!!! I don't have to much to day, just a question about the commented lines :-)
> 
> In docs/_sass/position.scss <https://github.com/aureamunoz/dekorate/pull/1#discussion_r786753031>:
> 
> > @@ -179,7 +179,7 @@ $position-large-margin-l:                               50px !default;
>  
>  .uk-position-absolute { position: absolute !important; }
>  
> -.uk-position-fixed { position: fixed !important; }
> +.uk-position-fixed { /* position: fixed !important; */ }
> Shouldn't we just remove the commented part?
> 
> .uk-position-fixed { /* position: fixed !important; */ }
> In docs/_sass/width.scss <https://github.com/aureamunoz/dekorate/pull/1#discussion_r786753727>:
> 
> > @@ -192,7 +192,8 @@ $width-2xlarge-width:                            750px !default;
>      .***@***.*** { width: unquote('calc(100% * 2 / 3.001)'); }
>  
>      /* Quarters */
> -    .***@***.*** { width: 25%; }
> +/*  .***@***.*** { width: 25%; } */
> +/*  .***@***.*** { width: 25%; } */
>      .***@***.*** { width: 75%; }
> Same remark, can we remove the commented part or it is used for something?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/aureamunoz/dekorate/pull/1#pullrequestreview-855417948>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAHCXEEWNSMZKE2QCINI2IDUWVSXJANCNFSM5MFNVV6A>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you authored the thread.
>
